app/src/main/java/com/locoquest/app/MainActivity.kt

```diff
@@ -104,19 +105,20 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
                     if (idToken != null) {
                         // Got an ID token from Google. Use it to authenticate
                         // with Firebase.
-                        val firebaseCredential = GoogleAuthProvider.getCredential(idToken, null)
-                        auth.signInWithCredential(firebaseCredential)
-                            .addOnCompleteListener(this) { task ->
-                                if (task.isSuccessful) {
-                                    // Sign in success, update UI with the signed-in user's information
-                                    Log.d(TAG, "signInWithCredential:success")
-                                    //TODO: hideSignInButton()
-                                    Log.d(TAG, "Got ID token.")
-                                } else {
-                                    // If sign in fails, display a message to the user.
-                                    Log.w(TAG, "signInWithCredential:failure", task.exception)
+                            val firebaseCredential = GoogleAuthProvider.getCredential(idToken, null)
+                            auth.signInWithCredential(firebaseCredential)
+                                .addOnCompleteListener(this) { task ->
+                                    if (task.isSuccessful) {
+                                        // Sign in success, update UI with the signed-in user's information
+                                        Log.d(TAG, "signInWithCredential:success")
+                                        //TODO: hideSignInButton()
+                                        Log.d(TAG, "Got ID token.")
+                                    } else {
+                                        // If sign in fails, display a message to the user.
+                                        Log.w(TAG, "signInWithCredential:failure", task.exception)
+                                    }
```

Comment:
Hmmmm, what changed here?  I've looked, and don't see anything.

Reply:
No change, this was probably me deleting something